card/lib/cardio.rb

```diff
@@ -50,26 +48,19 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
-      paths.add 'db/migrate_deck_cards', :with=>'db/migrate_cards'
+      add_gem_path 'tmp/set', :root => root
+      add_gem_path 'tmp/set_pattern', :root => root
```

Comment:
I get why you're doing this, but it renders the name `add_gem_path` very odd.

The issue is that if you only define the engine in the gem, then the paths root is in the gem for engine decks and in the app for app decks.  This means we really need to be explicit with every root.  (I had been thinking the engine would have a class for configuration in the deck, which would make the root there)

Perhaps we need to change the method to add_path and handle the engine v deck ambiguities there.